auto3dseg/algorithm_templates/segresnet2d/scripts/algo.py

```diff
@@ -29,6 +29,20 @@ def get_gpu_available_memory():
 
 
 class Segresnet2dAlgo(BundleAlgo):
+    def pre_check_skip_algo(self):
+        if self.data_stats_files is None:
+            return
```

Comment:
Should this raise an error? Something like `raise ValueError("data stats file is missing! Aborting.")`

Reply:
The pre_check is before generation and is not deciding if the whole autorunner should be stopped.  The error should be raised by the algo generation part if data_stats is missing.

Reply:
@heyufan1995 so who is calling/using this function "pre_check_skip_algo" ?

even it main code of autorunner should check for existence of  "self.data_stats_files", it's probably a good idea to raise and errors (abort) here as well, since datastats must exist at this point, right?

Reply:
I got your points. By the way, it would be good to add some docstring in this function as the `fill_template_config` about its scope of functionality. Thanks!